18 - ユーザ登録画面の作成

youmjww opened the pull request:
## 説明
- ユーザの登録画面の追加
- firebaseのエラーを対応している日本語のメッセージを付け加えた
- 要らないファイルを削除した
- ルータの設定を調整した

## Issue 
#18

youmjww committed:
18-eslintの設定を調整

問題
case文のにインデントを入れたらエラーが出ている。

解決策
エラーを出ないようにする。

youmjww committed:
18 - ユーザ登録画面を作成

問題
ユーザー登録用の画面が無かった

解決策
ユーザ登録用の画面を作った。
ルータにユーザ登録用の画面を追加した。
必要なパッケージをインストールした。

youmjww committed:
18 - 不要なファイルを削除

問題
TestComponent.jsというもう使われていないファイルがあった。

解決策
ファイルを削除した。

Dimda commented on Router.js:
```diff
@@ -1,17 +1,17 @@
 
       <Scene key="main">
-        <Scene key="home" component={Home} title="ホーム" />
+        <Scene initial back={false} key="home" component={Home} title="ホーム" />
```
多分 ` back={false}` なしでも動くと思う。

Commit:
18 - App以外のファイルを`src`に移動

問題
ソースコードに関連するファイルは全部ルートディレクトリにあって、
分かりにくい。

解決策
App以外のソースコードに関連するファイルを`src`のフォルダーに移動。

gucchii reviewed (approved):
しっかりと動いているとおもいます。
